cmd/kube_storage_monitor/app/server.go

```diff
@@ -38,25 +41,30 @@ func NewMonitorServerCommand() *cobra.Command {
 }
 
 func Run(mo *monitorOpt, stopCh <-chan struct{}) {
-	if len(mo.kube_storage_types) == 0 {
-		glog.Fatalf("kube-storage-types must be set")
+	if len(mo.kube_storage_types) == 0 && !mo.enable_node_watcher {
+		glog.Fatalf("either kube-storage-types or enable-node-watcher must be set")
 	}
-	// TODO(@NickrenREN): need to handle this more elegantly
-	// check if we support the storage types
-	err := checkStorageTypes(mo.kube_storage_types)
-	if err != nil {
-		glog.Errorf("check storage types error: %v", err)
-		os.Exit(1)
-	}
-	// run specific storage monitor
-	for _, sType := range mo.kube_storage_types {
-		switch sType {
-		// Add local_pv support at first
-		// If we get to support other storage types, need to add here
-		case "local_pv":
-			go local_pv.RunLocalPVMonitor()
+	if len(mo.kube_storage_types) > 0 {
```

Comment:
can I use both of the options? i.e. use `kube_storage_types` and `enable_node_watcher`

Reply:
no, here node watcher is deployed separately